docs/_guide/03-styling-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Styling templates
+slug: styling-templates
+---
+
+lit-html just renders HTML; you can style HTML created with lit-html like any other HTML, using CSS. How you apply styles to lit-html templates depends on how you're using it—for example, if you're using lit-html inside a component system like LitElement, you can follow the patterns used by that component system.
+
+In most cases you should **not** use lit-html bindings inside a style sheet. It's more efficient to use lit-html bindings to manipulate the `class` and `style` attributes.  
+
+lit-html provides two directives for manipulating an element's `class` and `style` attributes:
+
+*   [`classMap`](template-reference#classmap) sets classes on an element based on the properties of an object.
+*   [`styleMap`](template-reference#stylemap) sets the styles on an element based on a map of style properties and values.
+
+See [classMap](template-reference#classmap) and [styleMap](template-reference#stylemap) in the Template syntax reference.
+
+There are also some special considerations when using lit-html to render into a shadow root.
+
+## Rendering in shadow DOM
+
+When rendering into a shadow root, you usually want to add a stylesheet inside the shadow root to the template, to you can style the contents of the shadow root. 
+
+```js
+html`
+  <style>
+    :host { ... } 
+    .test { ... }
+  </style> 
+  <div class="test">...</div> 
+`;
+```
+
+
+### Don't use bindings in stylesheets 
```

Comment:
I would consider renaming this "Binding in Stylesheets"

Then recommending that in most cases you don't. In native Shadow DOM it's not that bad if the stylesheet is very small, for instance:

```js
html`
<style>
:host {
  --my-special-color: ${specialColor}
}
</style>
`
```

Although, if you're templated to do this, `styleMap` is probably better, so you could show that there.